### PR TITLE
LPD-33557 Obejct entries default language is following user's language instead of instance language

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-33557](https://liferay.atlassian.net/browse/LPD-33557)

[LPD-33557]: https://liferay.atlassian.net/browse/LPD-33557?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java [hunk 1]
```diff
@@ -3729,30 +3731,34 @@ public void testUpdateObjectEntry() throws Exception {
...
 			).put(
 				"localizedRichTextObjectFieldName",
 				"ar_SA localizedRichTextObjectFieldValue"
 			).put(
 				"localizedRichTextObjectFieldNameRawText",
 				"ar_SA localizedRichTextObjectFieldValue"
 			).put(
-				"localizedTextObjectFieldName", ""
+				"localizedTextObjectFieldName",
+				"en_US localizedTextObjectFieldValue1"
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:3744]: Hey @pedro-oliveira446 , could you please help me to make sure that we have other examples in the portal where makes sense to retrieve data like this, I mean a part of the data in one language and other part in other language.

If we have examples of it this pull request looks good and can be forward.

cc: @linolaoj

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
Hey @pedro-oliveira446 see my comment

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java (hunk 1 above)
Comment (pedro-oliveira446) [reply modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java:3744]: I discussed this behavior with Luis, and we based our approach on the Web Content component. Please see the video above for reference.


https://github.com/user-attachments/assets/4e196457-cbf4-43bb-9a66-2224239d8390

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:forms

**Comment by pedro-oliveira446** [comment]
ci:test:web-content

**Comment by pedro-oliveira446** [comment]
Hey @linolaoj the failures are not related !

**Comment by linolaoj** [comment]
ci:forward

**Comment by pedro-oliveira446** [comment]
ci:forward:force

**Pull request closed without merging**